specification/security/resource-manager/readme.md

```diff
@@ -60,6 +60,22 @@ directive:
     from: onPremiseIotSensors.json
     where: $.definitions.OnPremiseIotSensor
     reason: The resource is managed in a subscription level (instead of inside a resource group)
+  - suppress: TopLevelResourcesListByResourceGroup
+    from: ingestionSettings.json
+    where: $.definitions.IngestionSetting
+    reason: The resource is managed in a subscription level (instead of inside a resource group)
+  - suppress: OperationsAPIImplementation
+    from: ingestionSettings.json
+    where: $.paths
+    reason: There is no consistent implementation of operations api across existing versions
+  - suppress: SECRET_PROPERTY
+    from: ingestionSettings.json
+    where: $.definitions.IngestionSettingToken.properties.token
+    reason: Secrets are OK to return in a POST response.
+  - suppress: SECRET_PROPERTY
+    from: ingestionSettings.json
+    where: $.definitions.IngestionConnectionString.properties.value
+    reason: Secrets are OK to return in a POST response.
```

Comment:
@jianyexi please help check the suppression

Reply:
@kunchawla  you need to follow the swagger suppression process documented [here](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/85/Swagger-Suppression-Process) to add suppression

Reply:
Removed suppressions except for SECRET_PROPERTY. Per discussion, this didn't need a suppression process.